site/docs/v1/tech/client-libraries/go.adoc

```diff
@@ -15,69 +15,74 @@ Source Code:
 
 * https://github.com/FusionAuth/go-client
 
-Installation
+// TODO change this across all the client libraries
```

Comment:
Is this supposed to stay in here?

Reply:
Yes, because I changed the headers for the go client lib but not for the others. This is a reminder for me.